src/main/java/taskat6/DemoClass.java

```diff
@@ -0,0 +1,25 @@
+package taskat6;
+
+import java.util.Arrays;
+
+public class DemoClass {
+    public static void main(String[] args) {
+        System.out.println("Starting test.");
+        testClass();
+    }
+    static void testClass() {
```

Comment:
what is the method name? why is it static?

Reply:
made changes. it was very badly